mk/lint.mk

```
@@ -0,0 +1,15 @@
+.PHONY: install-pre-commit, install-golangci-lint, install-linter, lint, lint-all
+
+install-pre-commit:
+	pip install pre-commit
+
+install-golangci-lint:
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.47.2
```

```suggestion
	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell go env GOPATH)/bin v1.47.2
```
I think you mean the above.

Another suggestion would be to install into the ./release directory so the binary is used locally only by the repository. I suggest something like the below:

```suggestion
	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(GO_OUTPUT)" v1.47.2
```

`GO_OUTPUT` variable is set at `mk/variables.mk`, it is not a golang environment variable; maybe the name I used for this is not the best when it was added. This choice to install at ./release would require to override the PATH directory at `mk/variables.mk` such as:

```makefile
GO_OUTPUT ?= $(PROJECT_DIR)/release
PATH := $(GO_OUTPUT):$(PATH)
export PATH
```

> The rule `install-golangci-lint` as the above suggestion worked too in a macos system.

---

I think I changed this to how you suggested, but double check.